pages/index.js

```diff
@@ -1,21 +1,24 @@
+
 import { useEffect, useState } from "react";
 import { DataTable } from "primereact/datatable";
 import { Column } from "primereact/column";
 import { weatherTemplate, getWeatherIndex } from "../components/weatherTemplate";
+import { basePath } from "../next.config.js";
 
 
 export default function Home() {
   const [loading, setLoading] = useState(true);
   const [jobs, setJobs] = useState([]);
   const [rows, setRows] = useState([]);
   const [expandedRows, setExpandedRows] = useState([]);
+  const [keepSearch,     setKeepSearch]     = useState(true);
 
   useEffect(() => {
     const fetchData = async () => {
       let data = {};
 
       if (process.env.NODE_ENV === "development") {
-        data = (await import("../job_stats.json")).default;
+        data = (await import("../data/job_stats.json")).default;
```

Comment:
Please leave alone as per #20 

```suggestion
        data = (await import("../job_stats.json")).default;
```